### PR TITLE
Add Rabbit class and equals, hashCode, toString for all domain classes.

### PR DESCRIPTION
**Pull request by AlesyaZharnasek**

### CODE REVIEW

File: src/main/java/com/alesya/mafa/pet/Cat.java [hunk 1]
```diff
@@ -23,7 +24,28 @@ public void setHair(String hair) {
         this.hair = hair;
     }
 
+    public enum Breed {
+        ABYSSINIAN, BENGAL, BIRMAN, BOMBAY, BURMESE, CORNISH_REX
+    }
```
Comment (vitali-ivanou) [inline src/main/java/com/alesya/mafa/pet/Cat.java:59]: Please add for feature task - add nice printing of these breeds

File: src/main/java/com/alesya/mafa/pet/Cat.java [hunk 2]
```diff
@@ -23,7 +24,28 @@ public void setHair(String hair) {
...
+                "} " + super.toString();
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        if (!super.equals(o)) return false;
+        Cat cat = (Cat) o;
+        return Objects.equals(hair, cat.hair);
```
Comment (vitali-ivanou) [inline src/main/java/com/alesya/mafa/pet/Cat.java]: not redeable, please format this

File: src/main/java/com/alesya/mafa/pet/Dog.java [hunk 3]
```diff
@@ -25,6 +26,22 @@ public void setTail(String tail) {
...
+                "} " + super.toString();
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        if (!super.equals(o)) return false;
+        Dog dog = (Dog) o;
+        return Objects.equals(tail, dog.tail);
```
Comment (vitali-ivanou) [inline src/main/java/com/alesya/mafa/pet/Dog.java]: the same, please format this

File: src/main/java/com/alesya/mafa/pet/OperationInterface.java [hunk 4]
```diff
@@ -8,17 +8,19 @@
...
     private final static String HORSESHOE = "Input horseshoe: ";
     private final static String TAIL = "Input tail: ";
+    private final static String TYPE_OF_EARS = "Input type of ears: ";
 
     private ArrayList<Pet> storage = new ArrayList<>();
     private Scanner scanner;
 
     @Override
-    public Pet addPet() {
+    public Pet addPet(String BREED) {
```
Comment (vitali-ivanou) [inline src/main/java/com/alesya/mafa/pet/OperationInterface.java]: variable in upper case?

Comment (vitali-ivanou) [reply src/main/java/com/alesya/mafa/pet/OperationInterface.java]: Do you use this?

File: src/main/java/com/alesya/mafa/pet/OperationInterface.java [hunk 5]
```diff
@@ -97,23 +109,30 @@ public void setScanner(Scanner scanner) {
...
-        Cat cat = new Cat(addPet());
+        Cat cat = new Cat(addPet(CAT_BREED));
         System.out.println(HAIR);
         cat.setHair(scanner.nextLine());
         return cat;
     }
 
     private Horse createHorse() {
-        Horse horse = new Horse(addPet());
+        Horse horse = new Horse(addPet(BREED));
```
Comment (vitali-ivanou) [inline src/main/java/com/alesya/mafa/pet/OperationInterface.java]: why do you pass static field in parameters?

File: src/main/java/com/alesya/mafa/pet/StoragePets.java [hunk 6]
```diff
@@ -6,5 +6,6 @@ public static void main(String[] args) {
         MainInterface mainInterface = new MainInterface();
 
         mainInterface.run();
+
```
Comment (vitali-ivanou) [inline src/main/java/com/alesya/mafa/pet/StoragePets.java]: ?

**Review by vitali-ivanou — CHANGES_REQUESTED** [review]

File: src/main/java/com/alesya/mafa/pet/StoragePets.java (hunk 6 above)
Comment (AlesyaZharnasek) [reply src/main/java/com/alesya/mafa/pet/StoragePets.java]: done

File: src/main/java/com/alesya/mafa/pet/Cat.java (hunk 1 above)
Comment (AlesyaZharnasek) [reply src/main/java/com/alesya/mafa/pet/Cat.java:59]: done

File: src/main/java/com/alesya/mafa/pet/Cat.java (hunk 2 above)
Comment (AlesyaZharnasek) [reply src/main/java/com/alesya/mafa/pet/Cat.java]: done

File: src/main/java/com/alesya/mafa/pet/Dog.java (hunk 3 above)
Comment (AlesyaZharnasek) [reply src/main/java/com/alesya/mafa/pet/Dog.java]: done

File: src/main/java/com/alesya/mafa/pet/OperationInterface.java (hunk 4 above)
Comment (AlesyaMafa) [reply src/main/java/com/alesya/mafa/pet/OperationInterface.java]: updated

File: src/main/java/com/alesya/mafa/pet/OperationInterface.java (hunk 5 above)
Comment (AlesyaMafa) [reply src/main/java/com/alesya/mafa/pet/OperationInterface.java]: updated

File: src/main/java/com/alesya/mafa/pet/OperationInterface.java [hunk 7]
```diff
@@ -8,21 +8,25 @@
...
     @Override
-    public Pet addPet() {
+    public Pet addPet(String typePet) {
         Pet pet = new Pet();
         System.out.println(NAME);
         pet.setName(scanner.nextLine());
-        System.out.println(BREED);
+        if (typePet.equals("cat")) {
+            System.out.println(CAT_BREED);
+        } else System.out.println(BREED);
```
Comment (vitali-ivanou) [inline src/main/java/com/alesya/mafa/pet/OperationInterface.java]: format please

File: src/main/java/com/alesya/mafa/pet/OperationInterface.java [hunk 8]
```diff
@@ -97,23 +111,30 @@ public void setScanner(Scanner scanner) {
     }
 
     private Cat createCat() {
-        Cat cat = new Cat(addPet());
+        Cat cat = new Cat(addPet("cat"));
```
Comment (vitali-ivanou) [inline src/main/java/com/alesya/mafa/pet/OperationInterface.java]: change at all approach

**Review by vitali-ivanou — CHANGES_REQUESTED** [review]

**Pull request merged**